src/routes/LandingPage/EventsBanner/EventsBanner.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
```

Comment:
`eslint-disable` is great for developing stuff (i usually have it on too) but we do need to remove this line before merging into main

Reply:
All good, will get it changed